libs/langgraph/langgraph/pregel/loop.py

```diff
@@ -433,6 +459,7 @@ def __enter__(self) -> Self:
         self.status = "pending"
         self.step = self.checkpoint_metadata["step"] + 1
         self.stop = self.step + self.config["recursion_limit"] + 1
+        self.checkpoint_previous_versions = {}
```

Comment:
this should be `self.checkpoint_previous_versions = self.checkpoint['channel_versions']`